google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/WriteChannelConfiguration.java

```diff
@@ -274,6 +290,16 @@ public Builder setDecimalTargetTypes(List<String> decimalTargetTypes) {
       return this;
     }
 
+    public Builder setConnectionProperties(List<ConnectionProperty> connectionProperties) {
```

Comment:
not crucial, but if we use lombok, it covers some boiler plate code for getters and setters:
https://projectlombok.org/features/GetterSetter